index.js

```diff
@@ -9,13 +9,18 @@
     } else {
       var P = Promise || global.Promise
```

Comment:
doesn't this line break if `Promise` is not declared? (Also in master) Why would it be declared but falsey?